packages/web/src/app/web-player/ServerReduxProvider.tsx

```diff
@@ -0,0 +1,40 @@
+import { ReactElement } from 'react'
+
+import { Kind } from '@audius/common/src/models/Kind'
+import {
+  cacheTracksReducer,
+  cacheCollectionsReducer,
+  cacheUsersReducer
+} from '@audius/common/src/store/cache'
+import { asCache } from '@audius/common/src/store/cache/reducer'
+import { Provider } from 'react-redux'
+import { combineReducers, createStore } from 'redux'
+import { PartialDeep } from 'type-fest'
+
+import { AppState } from 'store/types'
+
+type ServerReduxProviderProps = {
+  children: ReactElement
+  initialState: PartialDeep<AppState>
+}
+
+const reducers = {
+  // @ts-ignore
+  users: asCache(cacheUsersReducer, Kind.TRACKS),
+  // @ts-ignore
+  tracks: asCache(cacheTracksReducer, Kind.TRACKS),
+  // @ts-ignore
+  collections: asCache(cacheCollectionsReducer, Kind.COLLECTIONS)
+}
```

Comment:
ah i see. making a mental note bc I was using the page reducers as well before, but i guess they arent strictly necessary

Reply:
I'm actually going to add those in after this is merged!

Reply:
nice